tests/phpunit/api/v3/JobTest.php

```diff
@@ -295,14 +295,42 @@ public function testCallDisableExpiredRelationships() {
       'end_date' => 'yesterday',
     ]);
     $relationshipID = $result['id'];
-    $this->assertEquals('Hooked', $result['values'][$relationshipID]['description']);
-    $this->callAPISuccess($this->_entity, 'disable_expired_relationships', []);
+    $this->assertEquals('Go Go you good thing', $result['values'][$relationshipID]['description']);
```

Comment:
Why is this change  needed - it looks like the test ensures the hook works & with this change no longer does

Reply:
Eileen this test was about the creation of relationship with certain parameters and effect of schedule job on that relationship. So previously the relationship was getting created with the dates being in past and is_active 1 but now with the past dates the is_active is converted to 0. And the description gets changed based on the is_active param in the hook and I have added one new test as well to check for the changed logic